pyproject.toml

```diff
@@ -18,7 +18,7 @@ description-file = "README.md"
 #        `typing.NoReturn` function signature support. (Also, many other `typing` module
 #        items were only introduced post-release of Python 3.5 and 3.6 and version
 #        restrictions on these versions ensure that those are all available as well.)
-requires-python = ">=3.5.4,!=3.6.0,!=3.6.1,!=3.7.0,!=3.7.1"
+requires-python = ">=3.6.2,!=3.7.0,!=3.7.1"
```

Comment:
Please also update the comment above.

Reply:
Can you clarify?

Reply:
Just above the part shown in the GitHub diff, there is some text that talks about the various limitations of the typings modules in Python 3.5, 3.6 and 3.7. The mentionings about 3.5 should be dropped since they no longer apply.